README.md

```diff
@@ -125,6 +125,58 @@ litgpt chat \
   --checkpoint_dir out/phi-2-lora/final
 ```
 
+
+&nbsp;
+
+### Deploy an LLM
+
+Deploy a LitGPT model using [LitServe](https://github.com/Lightning-AI/litserve):
+
+```python
+from litserve import LitAPI, LitServer
+...
+
+# STEP 1: DEFINE YOUR MODEL API
+class SimpleLitAPI(LitAPI):
+
+    def setup(self, device):
+        # Setup the model so it can be called in `predict`.
+        repo_id = "microsoft/phi-2"
+        checkpoint_dir = Path(f"checkpoints/{repo_id}")
+        ...
+ 
+
+    def decode_request(self, request):
+        # Convert the request payload to your model input.
+        prompt = request["prompt"]
+        ...
```

Comment:
William has said in the past that examples with `...` should be avoided as they are not runnable.

Could we instead have the actual deploy code you have in deploy.md?

It might be worth double checking what he prefers about this

Reply:
Yeah, let's use the full example here. It's easier to remove than to add. I.e., if someone reviews it, it should be easy for the reviewer to replace the verbose parts with `...` if desired.